docs/design/StratisSoftwareDesign.lyx

```diff
@@ -1141,518 +1110,5 @@ Journald interactions
-n of its replacement, by getting stratis to pvmove all the related volumes
- off of a dev.
- This could be built into "stratis dev remove" as a UI matter, by removing
- the ", if unused..." condition.
-\end_layout
-
-\end_inset
-
-
-\end_layout
-
-\begin_layout Verbatim
-stratis dev make-spare <pool_name> <dev>
-\end_layout
-
-\begin_layout Verbatim
-# Makes a blockdev into a spare blockdev, potentially moving
-\end_layout
-
-\begin_layout Verbatim
-# live data off of it.
- This may take a while, or fail if there
-\end_layout
-
-\begin_layout Verbatim
-# isn't enough available space in other blockdevs.
-\end_layout
-
-\begin_layout Verbatim
-
-\end_layout
-
-\begin_layout Verbatim
-stratis dev remove [-f] <pool_name> <dev> [<dev...]
-\end_layout
-
-\begin_layout Verbatim
-# Removes a blockdev from the pool, if unused or spare.
-\end_layout
-
-\begin_layout Verbatim
-# -f needed if would degrade a redundant pool
-\end_layout
-
-\begin_layout Section
-\begin_inset CommandInset label
-LatexCommand label
-name "sec:Dbus-API"
-
-\end_inset
-
-Dbus API
-\end_layout
-
-\begin_layout Standard
-All methods return at least a pair, consisting of a non-negative integer
- return code and a message string.
- A method may return more values; these precede the return code.
-\begin_inset Note Comment
-status collapsed
-
-\begin_layout Plain Layout
-houldn't all the objects have some type of UUID?
-\end_layout
-
-\end_inset
-
-
-\begin_inset Note Comment
-status collapsed
-
-\begin_layout Plain Layout
-If you utilize the standard dbus interfaces you can leverage ObjectManager
- and it's signals for objects being created and removed.
- You can also leverage the properties interface to get signals when properties
- of the objects change as well.
-\end_layout
-
-\end_inset
-
-
-\begin_inset Note Comment
-status collapsed
-
-\begin_layout Plain Layout
-Have you thought about monitoring issues? Beyond just logging, it would
- be good to have a mechanism to query cache hit rates, I/O amounts/speeds,
- identity of all constituent devices, whatever data might be available from
- the lower level md/kernel stats.
-\end_layout
-
-\end_inset
-
-
-\end_layout
-
-\begin_layout Subsection
-Manager
-\end_layout
-
-\begin_layout Subsubsection
-Overview
-\end_layout
-
-\begin_layout Standard
-Manager is the top-level interface.
- It makes available meta-information about stratisd, high-level methods,
- and general-purpose accessor methods.
-\end_layout
-
-\begin_layout Subsubsection
-Introspection Data
-\end_layout
-
-\begin_layout Standard
-\begin_inset Box Frameless
-position "t"
-hor_pos "c"
-has_inner_box 1
-inner_pos "t"
-use_parbox 0
-use_makebox 0
-width "100col%"
-special "none"
-height "1in"
-height_special "totalheight"
-thickness "0.4pt"
-separation "3pt"
-shadowsize "4pt"
-framecolor "black"
-backgroundcolor "none"
-status collapsed
-
-\begin_layout Plain Layout
-\begin_inset CommandInset include
-LatexCommand verbatiminput
-filename "manager_introspection.txt"
```

Comment:
If you are removing all references to the file, you might as well remove the file as well.